spine-ts/spine-player/example/assets/mix-and-match-pro.atlas

```diff
@@ -0,0 +1,370 @@
+mix-and-match-pro.png
```

Comment:
Rename the `mixAndMatch` folder to `audio` and move it to `examples/mix-and-match/`. Add the audio events to the `examples/mix-and-match/mix-and-match-pro.spine` Spine project file, make sure the Audio Files node has its path set to `./audio`. Modify `examples/export/runtimes.sh` to copy over the `.json`, `.atlas`, `.png` files to `spine-ts/player/example/assets/`, and also make it copy everything from `examples/mix-and-match/audio` to `spine-ts/player/example/assets/audio`. Remove `example-audio.html` and instead add a third player to `example.html` that loads the mix-and-match example.

To explain why: we want a single source of truth for the Spine projects that are used in all Spine Runtimes examples. We store them in `examples/`. When one of these Spine projects changes, or when we release a new Spine Editor major.minor version, we run `examples/export/export.sh`. That will export all Spine projects to `.json`, `.skel`, and `.atlas` files, stored in each project's respective `$PROJECT_NAME/export` folder. We then run `examples/export/runtimes.sh` to sprinkle those files around the example projects of each runtime.